agent/outbound_test.go

```diff
@@ -371,3 +375,51 @@ func Test_addAppProjectDeletionToQueue(t *testing.T) {
 		require.False(t, a.projectManager.IsManaged("test-project"))
 	})
 }
+
+func Test_addClusterCacheInfoUpdateToQueue(t *testing.T) {
+	a := newAgent(t)
+	a.remote.SetClientID("agent")
+	a.emitter = event.NewEventSource("principal")
+
+	miniRedis, err := miniredis.Run()
+	require.NoError(t, err)
+	require.NotNil(t, miniRedis)
+	defer miniRedis.Close()
+	err = WithRedisHost(miniRedis.Addr())(a)
+	require.NoError(t, err)
+
+	// Create the required Redis secret
+	secret := &corev1.Secret{
+		ObjectMeta: v1.ObjectMeta{Name: common.RedisInitialCredentials},
+		Data:       map[string][]byte{common.RedisInitialCredentialsKey: []byte("password123")},
```

Comment:
You may want to use a randomly generated uuid, rather than `password123`. This is a test, so it's unlikely to be a security issue, but it's very easy to just use a random value.